### PR TITLE
VTOrc Documentation

### PR DESCRIPTION
**Pull request by GuptaManan100**
### Description

This PR does the documentation for the recent PRs - 
- https://github.com/vitessio/vitess/pull/11356
- https://github.com/vitessio/vitess/pull/11370
- https://github.com/vitessio/vitess/pull/11263

This PR adds the documentation for upgrade instructions, flags available in VTOrc and more.

### CODE REVIEW

File: content/en/docs/15.0/user-guides/configuration-basic/vtorc.md [hunk 1]
```diff
@@ -1,46 +1,133 @@
...
 ---
 
-`vtorc` is a fork of the [Orchestrator](https://github.com/openark/orchestrator) customized to run as a Vitess component. It is available as an experimental feature. It still needs thorough testing, and more features need to be added.
+`VTOrc` is the automated fault detection and repair tool of Vitess. It started off as a fork of the [Orchestrator](https://github.com/openark/orchestrator), which was then custom-fitted to the Vitess use-case running as a Vitess component. It has reached general availablity with this release of Vitess.
 
-In order to configure `vtorc`, you have to make the following changes to `vttablet`:
+In order to configure `VTOrc`, you have to make the following changes to `vttablet`:
 
-* Add `--disable_active_reparents=true`: This part will prevent vttablet from fixing replication, and will rely on vtorc instead.
+* Add `--disable_active_reparents=true`: This part will prevent vttablet from fixing replication, and will rely on VTOrc instead.
```
Comment (deepthi) [inline content/en/docs/15.0/user-guides/configuration-basic/vtorc.md:10]: Is this still correct?
We introduced `disable-replication-manager` for this..
Aren't there side-effects to setting this flag? Like you can't do a PRS?
Or is it the expectation that you never have to do a PRS?

File: content/en/docs/15.0/reference/programs/vtorc.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  --port 15000 \
+  --recovery-period-block-duration "10m" \
+  --instance-poll-time "1s" \
+  --topo-information-refresh-duration "30s" \
+  --alsologtostderr
+```
+
+## Options
+
+The following global options apply to `VTOrc`:
```
Comment (deepthi) [inline content/en/docs/15.0/reference/programs/vtorc.md]:
```suggestion
The following command line options apply to `VTOrc`:
```

**Review by deepthi — APPROVED** [review]
You may merge this once comments are addressed.
If any further changes are needed we can make them in another PR.

File: content/en/docs/15.0/user-guides/configuration-basic/vtorc.md (hunk 1 above)
Comment (GuptaManan100) [reply content/en/docs/15.0/user-guides/configuration-basic/vtorc.md:10]: `disable-replication-manager` only disables the replication manager. `disable_active_reparents` also stops the tablet from initializing its own replication. Neither of these flags on the `vttablet` has any impact on PRS. 
`disable_active_reparents` prevents PRS if it is specified on the `vtctld` server.

Comment (GuptaManan100) [reply content/en/docs/15.0/user-guides/configuration-basic/vtorc.md:10]: They can use either, neither, both, everything works with VTOrc.

**Pull request merged**